src/ui/views/session/entity-selected/EntitySelected.css

```diff
@@ -0,0 +1,30 @@
+.EntitySelected {
+  position: absolute;
+  top: 50vh;
+  left: 50vw;
+  margin-top: 15px;
+  margin-left: 15px;
+  padding: 5px 8px;
+  background-color: var(--bg-world-overlay);
+  border-radius: 0px var(--br-xs) var(--br-xs) var(--br-xs);
+  color: white;
+  opacity: 1;
+  gap: var(--sp-md);
+}
+
+.MouseIcon {
+  margin-bottom: -2px;
+  width: 12px;
+  height: 12px;
+}
+
+.BoxedKey {
```

Comment:
The boxed key should be a bit bigger. Our minimum font size should never be smaller than 10px and even so only in very rare cases such as this.